libr/Makefile

```diff
@@ -42,7 +40,30 @@ libr.${EXT_SO}:
 	rm -rf .libr/tcc # WHY
 	$(CC) $(LDFLAGS) -shared -dynamic -arch arm64 -o libr.${EXT_SO} .libr/*/*.o ../shlr/gdb/lib/libgdbr.a
 
-include Makefile-ar.mk
+STATIC_LIBS:=$(wildcard */libr_*.${EXT_AR}) $(shell find ../shlr -name '*.${EXT_AR}')
+libr.${EXT_AR}: ${STATIC_LIBS}
+	rm -rf .libr
+	mkdir .libr
+	cd .libr ; \
+		for FILE in ${STATIC_LIBS} ; do \
+			BASE=$$(basename $${FILE} .a) ; \
+			PREVOBJ=none ; \
+			for OBJ in $$(ar t ../$${FILE} | sort) ; do \
+				if [ $${PREVOBJ} = $${OBJ} ] ; then \
+					N=$$(expr $${N} + 1) ; \
```

Comment:
you can use $$(($$N+1)), its shorter and posix too

Reply:
I thought that was a bashism.

Reply:
it is not